packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -346,6 +347,7 @@ export class ChatContext {
   ): (...args: Args) => R {
     return (...args: Args): R => {
       try {
+        chatStatefulLogger.info(`Stateful client target function called: ${target}`);
```

Comment:
The above comment says `Calling stateful client`
and this comment says `Stateful client`
Do we want to make it consistent?